features/mbedtls/targets/TARGET_NUVOTON/TARGET_M480/sha/sha1_alt.h

```diff
@@ -16,12 +16,6 @@
 #ifndef MBEDTLS_SHA1_ALT_H
 #define MBEDTLS_SHA1_ALT_H
 
-#if !defined(MBEDTLS_CONFIG_FILE)
-#include "config.h"
-#else
-#include MBEDTLS_CONFIG_FILE
-#endif
-
```

Comment:
Same comment as above regarding config.h

Reply:
@andresag01 `sha1.h` handles config check and it is included in front before `sha1_alt.h`. The commit https://github.com/ARMmbed/mbed-os/pull/4925/commits/1c44b41a199b2c85d5196401df005c146db52342 refines the config check code.